DataFormats/Detectors/GlobalTracking/src/RecoContainer.cxx

```diff
@@ -659,6 +659,14 @@ RecoContainer::GlobalIDSet RecoContainer::getSingleDetectorRefs(GTrackID gidx) c
   GlobalIDSet table;
   auto src = gidx.getSource();
   table[src] = gidx;
+  if (src == GTrackID::ITSTPCTRD) {
+    const auto& parent0 = getITSTPCTRDTrack<o2::trd::TrackTRD>(gidx);
+    const auto& parent1 = getTPCITSTrack(parent0.getRefGlobalTrackId());
+    table[GTrackID::ITSTPC] = parent0.getRefGlobalTrackId();
+    table[GTrackID::ITS] = parent1.getRefITS();
```

Comment:
@martenole please note that now the ITSTPC track can return not only `GTrackID::ITS` ref but also `GTrackID::ITSAB`, which is just a reference on the range of clusters matched to TPC track. So, you should fill the table as
https://github.com/AliceO2Group/AliceO2/blob/e4e31d67a9389b74a00583956891f60e510704ed/DataFormats/Detectors/GlobalTracking/src/RecoContainer.cxx#L683

Reply:
@martenole I will push these RecoContainer modifications in my PR since need them also for other stuff